pkg/local_object_storage/shard/shard.go

```diff
@@ -336,6 +342,28 @@ func (s *Shard) updateObjectCounter() {
 
 		s.cfg.metricsWriter.SetObjectCounter(physical, cc.Phy())
 		s.cfg.metricsWriter.SetObjectCounter(logical, cc.Logic())
+
+		cnrList, err := s.metaBase.Containers()
+		if err != nil {
+			s.log.Warn("meta: can't read container list", zap.Error(err))
+			return
+		}
+
+		var totalPayload uint64
+
+		for i := range cnrList {
+			size, err := s.metaBase.ContainerSize(cnrList[i])
```

Comment:
first of all i wanted to write my concern that these ops seems pretty heavyweight for `updateMetrics`. Then i looked up where the method is called - only in `Init`. I suggest to rename to `initMetrics`